src/components/AmountInput.tsx

```diff
@@ -0,0 +1,36 @@
+import clsx from 'clsx';
+
+type InputProps = {
+  label?: string;
+  error?: string;
+  isDisabled?: boolean;
```

Comment:
consider `disabled` like HTML input elements

Reply:
Sure I guess it makes sense in this case to make it congruent with the input element, normally I like sticking to the naming convention. Done.